programming/index.md

```diff
@@ -21,3 +21,9 @@ Our code is versioned using [Git](https://git-scm.com/). Git lets us collaborate
 
 - [Brief Introduction](https://try.github.io)
 - [Branching and Code Review](branching)
+
+## Unit Testing
+We use unit testing to help catch bugs in our code. This lets us quickly identify what parts of our code need to be fixed.
+
+- [Unit Testing in Python](http://pythontesting.net/framework/unittest/unittest-introduction/#overview)
```

Comment:
@octoflo Can you replace this example with one that uses Pytest rather than Unittest?

You could use https://docs.pytest.org/en/latest/getting-started.html or find another one